extensions/ql-vscode/src/extension.ts

```diff
@@ -977,27 +971,8 @@ async function activateWithInstalledDistribution(
   const testRunner = new TestRunner(dbm, cliServer);
   ctx.subscriptions.push(testRunner);
 
-  let testManager: TestManagerBase | undefined = undefined;
-  if (isCanary()) {
```

Comment:
So previously, only our canary users were using this feature? And with this change, all users will be using it. It's probably safe since most users of the testing feature are internal and therefore using canary.

Reply:
Yes.